src/main/java/thePackmaster/cards/psychicpack/MalleableFire.java

```diff
@@ -0,0 +1,50 @@
+package thePackmaster.cards.psychicpack;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.actions.common.MakeTempCardInDrawPileAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import thePackmaster.actions.EasyXCostAction;
+import thePackmaster.actions.upgradespack.SuperUpgradeAction;
+import thePackmaster.cards.AbstractPackmasterCard;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+import static thePackmaster.util.Wiz.*;
+
+public class MalleableFire extends AbstractPackmasterCard {
+    public final static String ID = makeID("MalleableFire");
+    // intellij stuff attack, enemy, rare, 9, 3, , , , 
+
+    public MalleableFire() {
+        super(ID, -1, CardType.ATTACK, CardRarity.RARE, CardTarget.ENEMY);
+        baseDamage = 9;
+        exhaust = true;
+        this.cardsToPreview = new MoldedFire();
+    }
+
+    public void use(AbstractPlayer p, AbstractMonster m) {
+        addToBot(new EasyXCostAction(this,
+                (x, params)->{
+                    MoldedFire fire = new MoldedFire(x);
+                    for (int i = 0; i < params[0]; ++i) {
+                        SuperUpgradeAction.silentForceUpgrade(fire);
+                    }
+
+                    addToTop(new MakeTempCardInDrawPileAction(fire, 1, true, true));
```

Comment:
works because it copies over numbers and the action only cares about timesUpgraded, but if you use dual wield or something on it the preview card won't look right so i'll Fix That